Abdullah Taqi QA Excercise Submission

abdullahtaqi opened the pull request:
Hi, this is my submission for the QA excercise. Thank you

hbahuguna2 commented:
Hi Abdullah, 

There are some basic issues that need to be fixed. I'm getting the following error when running `mvn clean install` on the project:

[ERROR] org.apache.maven.surefire.testset.TestSetFailedException: Suite file /Users/hbahuguna/github/take-home-webdriver-test/abdullah/take-home-webdriver-test/TakeHomeWebdriverTest/C:/Users/abdul/eclipse-workspace/TakeHomeWebdriverTest/src/test/java/resources/THWDTtestng.xml is not a valid file

There should be at least one Page Object used which is not the case. It would be better to use Page Objects. And also the tests should be platform agnostic. 

Thanks!

abdullahtaqi commented:
Hi, I fixed some of the problems. Please read the README.md

FIX: I think the reason the build was failing was because the file path in the pom.xml for the suiteXmlFile was specified to my computer so I changed it to src\test\java\resources\THWDTtestng.xml. I tested it on my other computer and maven install worked. I think it should work now if you do maven install.

FIX: I only had enough time to make one Page Object for the Login Page which is used in the Login Success and Login Failure Tests.
